src/main/java/com/epam/training/logic/ChristmasGiftLogic.java

```diff
@@ -0,0 +1,11 @@
+package com.epam.training.logic;
+
+import com.epam.training.model.ChristmasGifts;
+
+public interface ChristmasGiftLogic {
+
+    public double calculateGiftsPrice(ChristmasGifts christmasGift);
```

Comment:
gift word is extract, just calculatePrice

Reply:
@FilippStankevichTR Done.
Please, see the changes here: https://github.com/AIniuseva/epam-oop-task/pull/1/files#diff-c29a8cd12a819c33bcb0402cacb3e5f78ca36dd4cd2c6ffb8615b060dc757ca1R7-R9